src/plugins/analysis/file_system_metadata/view/file_system_metadata.html

```diff
@@ -28,9 +28,9 @@
                     <td class="text-left" style="width: 25%">{{ metadata.creation_time | nice_unix_time | safe }}</td>
                 </tr>
                 <tr>
-                    <td class="text-right">permissions</td>
+                    <td class="text-right">file mode</td>
```

Comment:
in most instances i've seen, `filemode` is written as one word.
```suggestion
                    <td class="text-right">filemode</td>
```